src/NuGet.Core/NuGet.Commands/SignCommand/SignCommandRunner.cs

```diff
@@ -85,6 +86,37 @@ public class SignCommandRunner : ISignCommandRunner
             return success ? 0 : 1;
         }
 
+        /// <summary>
+        /// Used to validate a user specified certificate.
+        /// </summary>
+        /// <param name="cert">Certificate to be validated</param>
+        private static void ValidateCertificate(X509Certificate2 cert)
+        {
+            if (!SigningUtility.CertificateContainsEku(cert, Oids.CodeSigningEkuOid))
+            {
+                throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture,
+                    "The following certificate cannot be used for signing a package as it does not have Code Signing enhanced key usage - {0}",
```

Comment:
Localize this string, remove `{0}`

See below for additional comments on using a trailing `{0}` to append new lines. This is making things much more complicated and error prone than it needs to be.